check.sh

```diff
@@ -7,5 +7,10 @@ if [ "${NO_CHANGELOG_LABEL}" = "true" ]; then
 else
     # a changelog check is required
     # fail if the diff is empty
+
+    >&2 echo "User-visible changes should come with an entry in the changelog. This behavior
+can be overridden by using the \"no changelog\" label, which is used for changes
+that are not user-visible."
+
```

Comment:
Thanks it is a good UX improvement, but I think the condition needs to go after the `git diff`, to only print if it has actually failed, not always when the label is missing.

Reply:
You're right, fixed it. :)